examples/docs_snippets/docs_snippets/guides/components/shell-script-component/2-shell-command-empty.py

```diff
@@ -1,21 +1,20 @@
+from dataclasses import dataclass
+
 from dagster import Definitions
 from dagster_components import (
     Component,
     ComponentLoadContext,
     Resolvable,
 )
 
+@dataclass(frozen=True)
 class ShellCommand(Component, Resolvable):
     """COMPONENT SUMMARY HERE.
 
     COMPONENT DESCRIPTION HERE.
     """
 
-    def __init__(
-        self,
-        # added arguments here will define yaml schema via Resolvable
-    ):
-        pass
+    # added attributes here will define yaml schema via Resolvable
```

Comment:
@schrockn the reason `__init__` was used in the `SimpleAssetComponent` was because our docs favored that approach. This updates the tutorial to promote `@dataclass(frozen=True)`.

Please let me know if you'd prefer to explain the difference between the two, and/or if you are in favor of this change. Thanks!